.travis.yml

```diff
@@ -0,0 +1,8 @@
+language: node_js
+node_js:
+- 12
+script:
+- npm run test:unit
+notifications:
+  slack:
+    secure: UbukNUk7BRG39FYgQLMC3vUpSSFdZQVJxNBxVe5R710Q3IN+j9aBXNYoUE9CdXFBHQPiFcVLoguiKGsW4jj5V6LqlEszo2LIybkbryxi0v3UzWIZ4doEmu5TVysEfNTyqyp9I0sc4+fD/GorIjuaPrt21B4ocXJlGkkpY18bsSJxBPVMSFkmEHGXuJASliyQswGV12DwsL2I6OIv471y/IqdTEEESwAF3JC79M/vnICUmmePO98F5bf2rSWKTKRj1pF1vOwcdn++NkqQEoVmdaPbim0ZILruhBlQnBYi8dOQ++LUZaOWX/GiIJaSvVWaRP+eeH0pfY+5GKfWMQDHEJ2e85t1jF1k1BLnh1GVvuKq2av/C0rVm8DATzxpIDa0cufQvosWcK6pwIeNLbMLDbuzhcj6UI9puvBsN+gDoPTiXpsiKOw/qQHXtP2OQbQQOo8x8cKJCqZTO/TBCmaRWnOPGMYVoBVgu9o2dfcl9qNKT/sjD+PY9hCaLuIwRbR5LT7cIX8jwy4MzMUM0hYttqaW7o27cg44lpy6Zj7+dv0LHdMZIpv1QedCgHeNVrmmFC2lxak/NWUWwllQXEr5NG+PPDzJqZh/k5/fb78scotu/EiItZCUftx85jHxzhmcwK/0M7wpTvUlSSHNfZKC+kOTpsSuQC2ZGbwpnBANoOQ=
```

Comment:
This should be configured in the Travis web interface, now anyone can post messages to our Slack channel 🙈

Reply:
from: https://docs.travis-ci.com/user/notifications/#configuring-slack-notifications

> Note: We highly recommend you encrypt this value if your .travis.yml is stored in a public repository:

This is the encrypted version, so....?

Reply:
Ah, right.. didn't see that